eva-accession-ws/src/main/java/uk/ac/ebi/eva/accession/ws/dto/DatasetAlleleResponse.java

```diff
@@ -0,0 +1,46 @@
+/*
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package uk.ac.ebi.eva.accession.ws.dto;
+
+import java.util.Map;
+
+public class DatasetAlleleResponse {
```

Comment:
what is this for? it's not actually used, right?

Reply:
It is one of the components of BeaconAlleleResponse.